boards/arm/bcm958402m2_a72/doc/index.rst

```diff
@@ -0,0 +1,59 @@
+.. _bcm958402m2_a72:
+
+Broadcom BCM958402M2_A72
+########################
+
+Overview
+********
+The Broadcom BCM958402M2_A72 board utilizes the Viper BCM58402_A72 SoC
+to provide support for PCIe offload engine functionality.
+
+Hardware
+********
+The bcm958402m2_a72 is a PCIe card with the following physical features:
```

Comment:
Nits: It would be nice to choose either `BCM958402M2_A72` or `bcm958402m2_a72`, and stick with it.